app/models/automated_message/ctc_experience_survey.rb

```diff
@@ -0,0 +1,25 @@
+module AutomatedMessage
+  class CtcExperienceSurvey
+    def self.name
+      'messages.surveys.ctc_experience'.freeze
+    end
+
+    def self.survey_link(client)
+      ctc_rejected_status = client.intake.tax_return(2020).status == "file_rejected" ? 'TRUE' : 'FALSE'
+
+      "https://codeforamerica.co1.qualtrics.com/jfe/form/SV_cHN2H3IWcxAEKPA?&ExternalDataReference=#{client.id}&ctcRejected=#{ctc_rejected_status}&expGroup=#{client.ctc_experience_survey_variant}"
```

Comment:
if im understanding this correctly, its the same survey and we just send a different query param if its rejected?

Reply:
i noticed the first query param has an & in front too `?&` -- likely doesn't affect anything, but i think the & is extraneous.